guides/common/modules/proc_searching-compliance-reports.adoc

```diff
@@ -10,49 +10,49 @@ Use the Compliance Reports search field to filter the list of available reports
 . Optional: To see a list of available search parameters, click the empty *Search* field.
 . Enter the search query in the *Search* field and click *Search*.
 The search query is case insensitive.
+* You can create complex queries with the following logical operators: `and`, `not` and `has`.
+For more information about logical operators, see {AdministeringDocURL}Supported_Operators_for_Granular_Search_admin[Supported Operators for Granular Search] in _{AdministeringDocTitle}_.
+* You cannot use regular expressions in a search query.
+However, you can use multiple fields in a single search expression.
+For more information about all available search operators, see {AdministeringDocURL}Supported_Operators_for_Granular_Search_admin[Supported Operators for Granular Search] in _{AdministeringDocTitle}_.
+* You can bookmark a search to reuse the same search query.
+For more information, see {AdministeringDocURL}Creating_Bookmarks_admin[Creating Bookmarks] in _{AdministeringDocTitle}_.
 
 .Search query examples
-Find all compliance reports for which more than five rules failed::
+====
+Find all compliance reports for which more than five rules failed:
```

Comment:
```suggestion
Find all compliance reports for which more than five rules failed:

```

I would expect an empty line between each paragraph and code block.

Same applies to lines 29,35,41,47,53

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214